com.unity.multiplayer.mlapi/Runtime/Core/NetworkManager.cs

```diff
@@ -1158,7 +1158,7 @@ internal void HandleIncomingData(ulong clientId, NetworkChannel networkChannel,
                 }
 
                 // Client tried to send a network message that was not the connection request before he was accepted.
-                if (PendingClients.ContainsKey(clientId) && (PendingClients[clientId].ConnectionState == PendingClient.State.PendingApproval || (PendingClients[clientId].ConnectionState == PendingClient.State.PendingConnection && messageType != NetworkConstants.CONNECTION_REQUEST)))
+                if (PendingClients.TryGetValue(clientId, out PendingClient client) && (client.ConnectionState == PendingClient.State.PendingApproval || client.ConnectionState == PendingClient.State.PendingConnection && messageType != NetworkConstants.CONNECTION_REQUEST))
```

Comment:
This line is jam packed. Especially now that there's an inline variable declaration, I'd suggest breaking these expressions into multiple lines